scripts/slimy-update-snap.sh

```diff
@@ -52,9 +52,12 @@ do_mount $old old
 
 add_overlay old new
 
+rm -rf new/lib/python3.6/site-packages/curtin
```

Comment:
What's your thoughts on doing probert also?  Does it get weird with the compiled libs?
If it's not too hard that would be nice.

Reply:
Hm. Yes compiled libs would be weird, although I guess it ought to be possible to replace the python code, which is all we usually care about. Maybe one for the next branch? :)